### PR TITLE
translate 2022-08-23-kubernetes-1.25-blog.md

### PR DESCRIPTION
**Pull request by ldsdsy**
I translated the 2022-08-23-kubernetes-1.25-blog.md. Please review it.

### CODE REVIEW

File: content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md [hunk 1]
```diff
@@ -0,0 +1,402 @@
+---
+layout: blog
+title: "Kubernetes v1.25: Combiner"
+date: 2022-08-23
+slug: kubernetes-v1-25-release
+---
+<!--
+---
+layout: blog
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md:9]: We usually remove the `---` in metadata comment.

File: content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md [hunk 2]
```diff
@@ -0,0 +1,402 @@
...
+
+<!--
+**Authors**: [Kubernetes 1.25 Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.25/release-team.md)
+-->
+**作者**: [Kubernetes 1.25 发布团队](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.25/release-team.md)
+
+<!--
+Announcing the release of Kubernetes v1.25!
+-->
+宣布 Kubernetes v1.25 的发版!
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md:24]:
```suggestion
宣布 Kubernetes v1.25 的发版！
```

File: content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md [hunk 3]
```diff
@@ -0,0 +1,402 @@
...
+
+PodSecurityPolicy was initially [deprecated in v1.21](/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/), and with the release of v1.25, it has been removed. The updates required to improve its usability would have introduced breaking changes, so it became necessary to remove it in favor of a more friendly replacement. That replacement is [Pod Security Admission](/docs/concepts/security/pod-security-admission/), which graduates to Stable with this release. If you are currently relying on PodSecurityPolicy, please follow the instructions for [migration to Pod Security Admission](/docs/tasks/configure-pod-container/migrate-from-psp/).
+-->
+## 新增内容（主要主题）
+### 移除 PodSecurityPolicy；Pod Security Admission 成长为 Stable {#pod-security-changes}
+
+PodSecurityPolicy 是在 [1.21 版本中被弃用](/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/)，到 1.25 版本被移除。
+因为提升其可用性的变更会带来破坏性的变化，所以有必要将其删除，以支持一个更友好的替代品。
+这个替代品就是 [Pod Security Admission](/docs/concepts/security/pod-security-admission/)，它在这个版本里成长为 Stable。
+如果你最近依赖于 PodSecurityPolicy，请参考 [Pod Security Admission 迁移说明](/docs/tasks/configure-pod-container/migrate-from-psp/)。
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md:74]:
```suggestion
这个替代品就是 [Pod Security Admission](/zh-cn/docs/concepts/security/pod-security-admission/)，它在这个版本里成长为 Stable。
如果你最近依赖于 PodSecurityPolicy，请参考 [Pod Security Admission 迁移说明](/zh-cn/docs/tasks/configure-pod-container/migrate-from-psp/)。
```

File: content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md [hunk 4]
```diff
@@ -0,0 +1,402 @@
...
+
+<!--
+### Ephemeral Containers Graduate to Stable
+
+[Ephemeral Containers](/docs/concepts/workloads/pods/ephemeral-containers/) are containers that exist for only a limited time within an existing pod. This is particularly useful for troubleshooting when you need to examine another container but cannot use `kubectl exec` because that container has crashed or its image lacks debugging utilities. Ephemeral containers graduated to Beta in Kubernetes v1.23, and with this release, the feature graduates to Stable.
+-->
+### Ephemeral Containers 成长为 Stable
+
+[Ephemeral Containers](/docs/concepts/workloads/pods/ephemeral-containers/) 是在现有的 pod 中存在有限时间的容器。
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md:83]:
```suggestion
[Ephemeral Containers](/zh-cn/docs/concepts/workloads/pods/ephemeral-containers/) 是在现有的 Pod 中存在有限时间的容器。
```

File: content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md [hunk 5]
```diff
@@ -0,0 +1,402 @@
...
+当你需要检查另一个容器，但因为该容器已经崩溃或其镜像缺乏调试工具不能使用 "kubectl exec" 时，它对故障排除特别有用。
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md:84]:
```suggestion
当你需要检查另一个容器，但因为该容器已经崩溃或其镜像缺乏调试工具不能使用 `kubectl exec` 时，它对故障排除特别有用。
```

File: content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md [hunk 6]
```diff
@@ -0,0 +1,402 @@
...
+
+<!--
+### Support for cgroups v2 Graduates to Stable
+
+It has been more than two years since the Linux kernel cgroups v2 API was declared stable. With some distributions now defaulting to this API, Kubernetes must support it to continue operating on those distributions. cgroups v2 offers several improvements over cgroups v1, for more information see the [cgroups v2](https://kubernetes.io/docs/concepts/architecture/cgroups/) documentation. While cgroups v1 will continue to be supported, this enhancement puts us in a position to be ready for its eventual deprecation and replacement.
+-->
+### 对 cgroups v2 的稳定支持
+
+自 Linux 内核 cgroups v2 API 宣布稳定以来，已经有两年多的时间了。随着一些发行版现在默认使用该 API，Kubernetes 必须支持它以继续在这些发行版上运行。
+cgroups v2 比 cgroups v1 提供了一些改进，更多信息见 [cgroups v2](https://kubernetes.io/docs/concepts/architecture/cgroups/) 文档。
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md:96]:
```suggestion
cgroups v2 比 cgroups v1 提供了一些改进，更多信息见 [cgroups v2](https://kubernetes.io/zh-cn/docs/concepts/architecture/cgroups/) 文档。
```

File: content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md [hunk 7]
```diff
@@ -0,0 +1,402 @@
...
+- 为 [Windows Operational Readiness](https://github.com/kubernetes-sigs/windows-operational-readiness) 创建了新的 GitHub 仓库
+
+<!--
+### Moved container registry service from k8s.gcr.io to registry.k8s.io
+
+[Moving container registry from k8s.gcr.io to registry.k8s.io](https://github.com/kubernetes/kubernetes/pull/109938) got merged. For more details, see the [wiki page](https://github.com/kubernetes/k8s.io/wiki/New-Registry-url-for-Kubernetes-\(registry.k8s.io\)), [annoucement](https://groups.google.com/a/kubernetes.io/g/dev/c/DYZYNQ_A6_c/m/oD9_Q8Q9AAAJ) was sent to the kubernetes development mailing list.
+-->
+### 将容器注册服务从 k8s.gcr.io 迁移至 registry.k8s.io
+[将容器注册服务从 k8s.gcr.io 迁移至 registry.k8s.io](https://github.com/kubernetes/kubernetes/pull/109938) 的 pr 已经被合并。
+更多细节参考 [wixi 页面](https://github.com/kubernetes/k8s.io/wiki/New-Registry-url-for-Kubernetes-\(registry.k8s.io\))，
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md:121]:
```suggestion
更多细节参考 [wiki 页面](https://github.com/kubernetes/k8s.io/wiki/New-Registry-url-for-Kubernetes-\(registry.k8s.io\))，
```

File: content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md [hunk 8]
```diff
@@ -0,0 +1,402 @@
...
+- [一致性测试](https://github.com/kubernetes/kubernetes/pull/108592) 增加了对 Windows 系统的支持
+- 为 [Windows Operational Readiness](https://github.com/kubernetes-sigs/windows-operational-readiness) 创建了新的 GitHub 仓库
+
+<!--
+### Moved container registry service from k8s.gcr.io to registry.k8s.io
+
+[Moving container registry from k8s.gcr.io to registry.k8s.io](https://github.com/kubernetes/kubernetes/pull/109938) got merged. For more details, see the [wiki page](https://github.com/kubernetes/k8s.io/wiki/New-Registry-url-for-Kubernetes-\(registry.k8s.io\)), [annoucement](https://groups.google.com/a/kubernetes.io/g/dev/c/DYZYNQ_A6_c/m/oD9_Q8Q9AAAJ) was sent to the kubernetes development mailing list.
+-->
+### 将容器注册服务从 k8s.gcr.io 迁移至 registry.k8s.io
+[将容器注册服务从 k8s.gcr.io 迁移至 registry.k8s.io](https://github.com/kubernetes/kubernetes/pull/109938) 的 pr 已经被合并。
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md:120]:
```suggestion
[将容器注册服务从 k8s.gcr.io 迁移至 registry.k8s.io](https://github.com/kubernetes/kubernetes/pull/109938) 的 PR 已经被合并。
```

Comment (ldsdsy) [reply content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md]: Thank you for your guidance, I have revised it.

File: content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md [hunk 9]
```diff
@@ -0,0 +1,400 @@
...
+
+<!--
+Announcing the release of Kubernetes v1.25!
+-->
+宣布 Kubernetes v1.25 的发版！
+
+<!--
+This release includes a total of 40 enhancements. Fifteen of those enhancements are entering Alpha, ten are graduating to Beta, and thirteen are graduating to Stable. We also have two features being deprecated or removed.
+-->
+这个版本总共包括 40 项增强功能。其中 15 项增强功能进入 Alpha，10 项进入 Beta，13 项进入 Stable。我们也废弃/删除了两个功能。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md:27]:
```suggestion
这个版本总共包括 40 项增强功能。其中 15 项增强功能进入 Alpha，10 项进入 Beta，13 项进入 Stable。
我们也废弃/移除了两个功能。
```
- `Remove` is suggested to be localized as 移除.
- Give a newline char in-between if a clause is long.

File: content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md [hunk 10]
```diff
@@ -0,0 +1,400 @@
...
+With this release we wish to honor the collaborative, open spirit that takes us from isolated developers, writers, and users spread around the globe to a combined force capable of changing the world. Kubernetes v1.25 includes a staggering 40 enhancements, none of which would exist without the incredible power we have when we work together.
+
+Inspired by our release lead's son, Albert Song, Kubernetes v1.25 is named for each and every one of you, no matter how you choose to contribute your unique power to the combined force that becomes Kubernetes.
+-->
+## 版本主题和徽标
+**Kubernetes 1.25: Combiner**
+
+{{< figure src="/images/blog/2022-08-23-kubernetes-1.25-release/kubernetes-1.25.png" alt="Combiner logo" class="release-logo" >}}
+
+Kubernetes v1.25 的主题是 _Combiner_，即组合器。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md:49]:
```suggestion
Kubernetes v1.25 的主题是 **Combiner**，即组合器。
```

File: content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md [hunk 11]
```diff
@@ -0,0 +1,400 @@
...
+如果你最近依赖于 PodSecurityPolicy，请参考 [Pod Security Admission 迁移说明](/zh-cn/docs/tasks/configure-pod-container/migrate-from-psp/)。
+
+<!--
+### Ephemeral Containers Graduate to Stable
+
+[Ephemeral Containers](/docs/concepts/workloads/pods/ephemeral-containers/) are containers that exist for only a limited time within an existing pod. This is particularly useful for troubleshooting when you need to examine another container but cannot use `kubectl exec` because that container has crashed or its image lacks debugging utilities. Ephemeral containers graduated to Beta in Kubernetes v1.23, and with this release, the feature graduates to Stable.
+-->
+### Ephemeral Containers 成长为 Stable
+
+[Ephemeral Containers](/zh-cn/docs/concepts/workloads/pods/ephemeral-containers/) 是在现有的 Pod 中存在有限时间的容器。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md:81]:
```suggestion
[临时容器](/zh-cn/docs/concepts/workloads/pods/ephemeral-containers/)是在现有的 Pod 中存在有限时间的容器。
```

File: content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md [hunk 12]
```diff
@@ -0,0 +1,400 @@
...
+当你需要检查另一个容器，但因为该容器已经崩溃或其镜像缺乏调试工具不能使用 `kubectl exec` 时，它对故障排除特别有用。
+Ephemeral containers 在 Kubernetes v1.23 中成长为 Beta，并在这个版本中，该功能成长为 Stable。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md:83]:
```suggestion
临时容器在 Kubernetes v1.23 中成长为 Beta，并在这个版本中，该功能成长为 Stable。
```

File: content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md [hunk 13]
```diff
@@ -0,0 +1,400 @@
...
+
+
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md:85]:
```suggestion

```

File: content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md [hunk 14]
```diff
@@ -0,0 +1,400 @@
...
+<!--
+### Support for cgroups v2 Graduates to Stable
+
+It has been more than two years since the Linux kernel cgroups v2 API was declared stable. With some distributions now defaulting to this API, Kubernetes must support it to continue operating on those distributions. cgroups v2 offers several improvements over cgroups v1, for more information see the [cgroups v2](https://kubernetes.io/docs/concepts/architecture/cgroups/) documentation. While cgroups v1 will continue to be supported, this enhancement puts us in a position to be ready for its eventual deprecation and replacement.
+-->
+### 对 cgroups v2 的稳定支持
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md:91]:
```suggestion
### 对 cgroups v2 的支持进入 Stable 阶段
```

File: content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md [hunk 15]
```diff
@@ -0,0 +1,400 @@
...
+
+自 Linux 内核 cgroups v2 API 宣布稳定以来，已经有两年多的时间了。随着一些发行版现在默认使用该 API，Kubernetes 必须支持它以继续在这些发行版上运行。
+cgroups v2 比 cgroups v1 提供了一些改进，更多信息见 [cgroups v2](https://kubernetes.io/zh-cn/docs/concepts/architecture/cgroups/) 文档。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md:94]:
```suggestion
cgroups v2 比 cgroups v1 提供了一些改进，更多信息参见 [cgroups v2](/zh-cn/docs/concepts/architecture/cgroups/) 文档。
```

File: content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md [hunk 16]
```diff
@@ -0,0 +1,400 @@
...
+
+<!--
+### Moved container registry service from k8s.gcr.io to registry.k8s.io
+
+[Moving container registry from k8s.gcr.io to registry.k8s.io](https://github.com/kubernetes/kubernetes/pull/109938) got merged. For more details, see the [wiki page](https://github.com/kubernetes/k8s.io/wiki/New-Registry-url-for-Kubernetes-\(registry.k8s.io\)), [annoucement](https://groups.google.com/a/kubernetes.io/g/dev/c/DYZYNQ_A6_c/m/oD9_Q8Q9AAAJ) was sent to the kubernetes development mailing list.
+-->
+### 将容器注册服务从 k8s.gcr.io 迁移至 registry.k8s.io
+[将容器注册服务从 k8s.gcr.io 迁移至 registry.k8s.io](https://github.com/kubernetes/kubernetes/pull/109938) 的 PR 已经被合并。
+更多细节参考 [wiki 页面](https://github.com/kubernetes/k8s.io/wiki/New-Registry-url-for-Kubernetes-\(registry.k8s.io\))，
+同时 [公告](https://groups.google.com/a/kubernetes.io/g/dev/c/DYZYNQ_A6_c/m/oD9_Q8Q9AAAJ) 已发送到 kubernetes 开发邮件列表。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md:120]:
```suggestion
同时[公告](https://groups.google.com/a/kubernetes.io/g/dev/c/DYZYNQ_A6_c/m/oD9_Q8Q9AAAJ)已发送到 kubernetes 开发邮件列表。
```

File: content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md [hunk 17]
```diff
@@ -0,0 +1,400 @@
...
+<!--
+### Improved Windows support
+
+- [Performance dashboards](http://perf-dash.k8s.io/#/?jobname=soak-tests-capz-windows-2019) added support for Windows
+- [Unit tests](https://github.com/kubernetes/kubernetes/issues/51540) added support for Windows
+- [Conformance tests](https://github.com/kubernetes/kubernetes/pull/108592) added support for Windows
+- New GitHub repository created for [Windows Operational Readiness](https://github.com/kubernetes-sigs/windows-operational-readiness)
+-->
+### 改善对 Windows 系统的支持
+- [性能仪表板](http://perf-dash.k8s.io/#/?jobname=soak-tests-capz-windows-2019) 增加了对 Windows 系统的支持
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md:107]:
```suggestion
- [性能仪表板](http://perf-dash.k8s.io/#/?jobname=soak-tests-capz-windows-2019)增加了对 Windows 系统的支持
```

File: content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md [hunk 18]
```diff
@@ -0,0 +1,400 @@
...
+更多细节参考 [wiki 页面](https://github.com/kubernetes/k8s.io/wiki/New-Registry-url-for-Kubernetes-\(registry.k8s.io\))，
+同时 [公告](https://groups.google.com/a/kubernetes.io/g/dev/c/DYZYNQ_A6_c/m/oD9_Q8Q9AAAJ) 已发送到 kubernetes 开发邮件列表。
+
+<!--
+### Promoted SeccompDefault to Beta
+
+SeccompDefault promoted to beta, see the tutorial [Restrict a Container's Syscalls with seccomp](https://kubernetes.io/docs/tutorials/security/seccomp/#enable-the-use-of-runtimedefault-as-the-default-seccomp-profile-for-all-workloads) for more details.
+-->
+### SeccompDefault 升级为 Beta
+SeccompDefault 升级为 Beta，更多细节参考教程 [用 seccomp 限制一个容器的系统调用](https://kubernetes.io/zh-cn/docs/tutorials/security/seccomp/#enable-the-use-of-runtimedefault-as-the-default-seccomp-profile-for-all-workloads) 。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md:128]:
```suggestion
SeccompDefault 升级为 Beta，
更多细节参考教程[用 seccomp 限制一个容器的系统调用](/zh-cn/docs/tutorials/security/seccomp/#enable-the-use-of-runtimedefault-as-the-default-seccomp-profile-for-all-workloads) 。
```

File: content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md [hunk 19]
```diff
@@ -0,0 +1,400 @@
...
+<!--
+### Promoted endPort in Network Policy to Stable
+
+Promoted `endPort` in [Network Policy](https://kubernetes.io/docs/concepts/services-networking/network-policies/#targeting-a-range-of-ports) to GA. Network Policy providers that support `endPort` field now can use it to specify a range of ports to apply a Network Policy. Previously, each Network Policy could only target a single port.
+
+Please be aware that `endPort` field **must be supported** by the Network Policy provider. If your provider does not support `endPort`, and this field is specified in a Network Policy, the Network Policy will be created covering only the port field (single port).
+-->
+### 网络策略中 endPort 升级为 Stable
+
+[网络策略](https://kubernetes.io/zh-cn/docs/concepts/services-networking/network-policies/#targeting-a-range-of-ports) 中的 `endPort` 已经迎来 GA 通用版。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md:139]:
```suggestion
[网络策略](/zh-cn/docs/concepts/services-networking/network-policies/#targeting-a-range-of-ports) 中的
`endPort` 已经迎来 GA 通用版。
```

File: content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md [hunk 20]
```diff
@@ -0,0 +1,400 @@
...
+支持 `endPort` 字段的网络策略提供程序，现在可使用该字段来指定端口范围来应用网络策略。在之前的版本中，每个网络策略只能指向单一端口。
+
+请注意，网络策略提供程序 **必须支持** `endPort` 字段。如果提供程序不支持 `endPort`，又在网络策略中指定了此字段，则会创建出仅覆盖端口字段（单端口）的网络策略。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md:142]:
```suggestion
请注意，网络策略提供程序 **必须支持** `endPort` 字段。
如果提供程序不支持 `endPort`，又在网络策略中指定了此字段，
则会创建出仅覆盖端口字段（单端口）的网络策略。
```
全文统改，长句在标点、中英之间折行

**Review by windsonsea — COMMENTED** [review]
https://kubernetes.io/zh-cn/docs/concepts/workloads/pods/ephemeral-containers/

File: content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md (hunk 20 above)
Comment (ldsdsy) [reply content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md]: Thank you for the suggestion, it has been reworked.

**Comment by tengqm** [comment]
/hold
I believe some more changes to this blog is coming in.

https://github.com/kubernetes/website/pull/36430

**Comment by ldsdsy** [comment]
/assign @tengqm 
#36430 has been merged and I have amended it accordingly, please take another look.

File: content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md [hunk 21]
```diff
@@ -0,0 +1,431 @@
...
+### Promoted endPort in Network Policy to Stable
+
+Promoted `endPort` in [Network Policy](https://kubernetes.io/docs/concepts/services-networking/network-policies/#targeting-a-range-of-ports) to GA. Network Policy providers that support `endPort` field now can use it to specify a range of ports to apply a Network Policy. Previously, each Network Policy could only target a single port.
+
+Please be aware that `endPort` field **must be supported** by the Network Policy provider. If your provider does not support `endPort`, and this field is specified in a Network Policy, the Network Policy will be created covering only the port field (single port).
+-->
+### 网络策略中 endPort 升级为 Stable
+
+[网络策略](/zh-cn/docs/concepts/services-networking/network-policies/#targeting-a-range-of-ports)中的 
+`endPort` 已经迎来 GA 通用版。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md:148]:
```suggestion
`endPort` 已经迎来 GA 正式发布。
```

File: content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md [hunk 22]
```diff
@@ -0,0 +1,431 @@
...
+支持 `endPort` 字段的网络策略提供程序，现在可使用该字段来指定端口范围来应用网络策略。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md:149]:
```suggestion
支持 `endPort` 字段的网络策略提供程序现在可使用该字段来指定端口范围，应用网络策略。
```

File: content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md [hunk 23]
```diff
@@ -0,0 +1,431 @@
...
+请注意，网络策略提供程序 **必须支持** `endPort` 字段。
+如果提供程序不支持 `endPort`，又在网络策略中指定了此字段，
+则会创建出仅覆盖端口字段（单端口）的网络策略。
+
+<!--
+### Promoted Local Ephemeral Storage Capacity Isolation to Stable
+The [Local Ephemeral Storage Capacity Isolation](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/361-local-ephemeral-storage-isolation) feature moved to GA. This was introduced as alpha in 1.8, moved to beta in 1.10, and it is now a stable feature. It provides support for capacity isolation of local ephemeral storage between pods, such as `EmptyDir`, so that a pod can be hard limited in its consumption of shared resources by evicting Pods if its consumption of local ephemeral storage exceeds that limit.
+-->
+### 本地临时容器存储容量隔离升级为 Stable
+[本地临时存储容量隔离功能](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/361-local-ephemeral-storage-isolation)已经迎来 GA 通用版。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md:161]:
```suggestion
[本地临时存储容量隔离功能](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/361-local-ephemeral-storage-isolation)已经迎来 GA 正式发布版本。
```

Comment (ldsdsy) [reply content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md]: Thank you for your suggestion, it has been amended.

Comment (windsonsea) [reply content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md]: > Thank you for your suggestion, it has been amended.

If you have modified all, please click the `Resolve conversation` button.

**Comment by ldsdsy** [comment]
Is there anything else that needs to be changed?

**Pull request merged**